webpack.server.config.js

```diff
@@ -0,0 +1,56 @@
+const webpack = require('webpack');
+const path = require('path');
+
+const ENV = process.env.NODE_ENV || 'development';
+
+const sConfig = {
+  name: 'server',
+  target: 'node',
+  entry: './server/server.js',
+  output: {
+    path: path.resolve(__dirname, 'dist'),
+    libraryTarget: 'commonjs2',
+    filename: 'server.js',
+    publicPath: '/',
+  },
+  module: {
+    rules: [{
+      enforce: 'pre',
+      test: /\.jsx?$/,
+      use: 'eslint-loader',
+      exclude: /node_modules/,
+    }, {
+      test: /\.jsx?$/,
+      exclude: /node_modules/,
+      use: 'babel-loader',
+    }, {
+      test: /\.scss$/,
+      use: [{
+        loader: 'css-loader/locals',
```

Comment:
We have to remove style loaders because the code generated depends on access to the DOM